examples/features/custom_reducers_mnist_pytorch/model_def.py

```diff
@@ -0,0 +1,153 @@
+"""
+This example shows how to use custom reducers with PyTorch.  In this example,
+the custom reducer is a per-class F1 score.  It is based on Determined's
+mnist_pytorch tutorial code.
+
+The parts of the file which differ from the tutorial are the PerClassF1Score class
+and the calls to wrap_reducer() and self.f1_score.update.
+"""
+
+from typing import Any, Dict, Sequence, Tuple, Union, cast
+
+import torch
+from torch import nn
+
+from layers import Flatten  # noqa: I100
```

Comment:
What's this comment here?

Reply:
It's for the linter, but it's copy/pasted from the original